pandas/tests/frame/indexing/test_setitem.py

```diff
@@ -407,6 +407,11 @@ def test_setitem_frame_duplicate_columns(self, using_array_manager):
             expected["C"] = expected["C"].astype("int64")
             # TODO(ArrayManager) .loc still overwrites
             expected["B"] = expected["B"].astype("int64")
+
+            mark = pytest.mark.xfail(
```

Comment:
what this about?

Reply:
the ArrayManager behavior is different for reasons i'm still chasing down